File: src/test/java/br/com/model/entities/classes/NovoPedidoTest.java
package br.com.model.entities.classes;

import org.junit.jupiter.api.Test;

import br.com.newObjects.model.entities.classes.NovoPedido;

public class NovoPedidoTest {
    @Test
    void testNovoPedido() {
        System.out.println(NovoPedido.novoPedido().toString());
    }
}

File: src/test/java/br/com/model/entities/classes/NovoPedidoTest.kt
package br.com.model.entities.classes

import br.com.newObjects.model.entities.classes.NovoPedido
import org.junit.jupiter.api.Test

class NovoPedidoTest {
    @Test
    fun testNovoPedido() {
        println(NovoPedido.novoPedido().toString())
    }
}
